app/scripts/utils/use-feedback-modal.tsx

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react';
+import { useVedaUI } from '$context/veda-ui-provider';
+/**
+ * Hook to access the feedback modal.
+ */
+export function useFeedbackModal() {
+  const { feedbackModalRevealed, setFeedbackModalRevealed } = useVedaUI();
```

Comment:
We cannot use layout provider, so I added these to VEDA UI Provider. but if there is any better place. willing to move this.